net/happygears/proto/NSGPythonSDK.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package net.happygears.proto.nsg_python_sdk;
+import "net/happygears/proto/NSG.proto";
+
+message ServiceInfo {
+  string script = 1;
+  string port = 2;
+}
+
+message StatusResp {
+  string status = 1;
+}
+
+message TagsAppRequest {
+  nsg.Device device = 1;
+}
+
+message KeyedTagList {
+  repeated string tags = 1;
```

Comment:
I am moving KeyedTagList and TagUpdateJournal to NSG (proto-oss), so you should not need to copy them here

Reply:
Okay, I will update the proto-java dependency and make necessary changes.